pkg/kubelet/dockershim/network/hostport/hostport_syncer.go

```diff
@@ -227,6 +229,9 @@ func (h *hostportSyncer) SyncHostports(natInterfaceName string, activePodPortMap
 	for port, target := range hostportPodMap {
 		protocol := strings.ToLower(string(port.Protocol))
 		hostportChain := hostportChainName(port, target.podFullName)
+		if utilnet.IsIPv6String(target.podIP) != h.iptables.IsIpv6() {
```

Comment:
same comment about multi-IP

Reply:
I've changed the approach so kubenet initiates one hostport_manager and hostport_syncer per IP family. If the pod IP doesn't match the IP familiy of the hostport_manager/syncer it simply skips it

Reply:
hostport syncer and manager only works with the IP family they have configured
hostport manager fails if the pod IP family is different.
hostport syncer only gets the portMaps from the Pods with the same IP family is configured, skipping the others